backend/main/src/Group/Router/GroupRouter.ts

```diff
@@ -19,6 +31,36 @@ export const groupRouter = (app: Hono) => {
 		return honoNotImplementedAdapter(c);
 	});
 
+	app.put("/groups/:groupId", async (c) => {
+		const groupId = c.req.param("groupId");
+		const { Action } = c.req.query();
```

Comment:
This is just a question. You don't need to modify it.
I wanna know the reason that you don't explicitly specify the query name like this.
```typescript
const Action = c.req.query("Action");
```

Reply:
There is no reason. But your way is more straightforward than my way. I'll fix it, thank you for pointing it out.